File: android/app/src/main/java/com/theoplayer/contentprotectionintegration/integration/azuredrm/AzureWidevineContentProtectionIntegrationFactory.java
package com.theoplayer.contentprotectionintegration.integration.azuredrm;

import com.theoplayer.android.api.contentprotection.ContentProtectionIntegration;
import com.theoplayer.android.api.contentprotection.ContentProtectionIntegrationFactory;
import com.theoplayer.android.api.source.drm.DRMConfiguration;

public class AzureWidevineContentProtectionIntegrationFactory implements ContentProtectionIntegrationFactory {
    @Override
    public ContentProtectionIntegration build(DRMConfiguration configuration) {
        return new AzureWidevineContentProtectionIntegration(configuration);
    }
}

File: android/app/src/main/java/com/theoplayer/contentprotectionintegration/integration/azuredrm/AzureWidevineContentProtectionIntegrationFactory.kt
package com.theoplayer.contentprotectionintegration.integration.azuredrm

import com.theoplayer.android.api.contentprotection.ContentProtectionIntegrationFactory
import com.theoplayer.android.api.source.drm.DRMConfiguration
import com.theoplayer.android.api.contentprotection.ContentProtectionIntegration

class AzureWidevineContentProtectionIntegrationFactory : ContentProtectionIntegrationFactory {
    override fun build(configuration: DRMConfiguration): ContentProtectionIntegration {
        return AzureWidevineContentProtectionIntegration(configuration)
    }
}